java/code/webapp/WEB-INF/pages/admin/multiorg/configuration.jsp

```diff
@@ -1,82 +1,78 @@
 <%@ taglib uri="http://rhn.redhat.com/rhn" prefix="rhn" %>
-<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
-<%@ taglib uri="http://struts.apache.org/tags-html" prefix="html" %>
-<%@ taglib uri="http://struts.apache.org/tags-bean" prefix="bean" %>
+  <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+    <%@ taglib uri="http://struts.apache.org/tags-html" prefix="html" %>
+      <%@ taglib uri="http://struts.apache.org/tags-bean" prefix="bean" %>
```

Comment:
Here and in the rest of this file, the formatting is wrong. It seems whatever autoformatter you're using doesn't recognize that these are references, not actual opening tags.

@cbbayburt Do you happen to know if we can autofix formatting in jsp files as well somehow conveniently? Would be nice to just have a linter do all of this automatically.